docs/conf/cincinnati/2018/sprints.rst

```diff
@@ -0,0 +1,22 @@
+:template: {{year}}/generic.html
+
+Sprints
+=======
+
+After the conference and unconference days, we will host three days of Open Help team sprints.
+Open Help has hosted sprints for major open source projects like GNOME, Mozilla, WordPress, and FreeBSD.
+
+The sprint days allow entire open source teams to come together and collaborate.
+Your team sets its own agenda, but we'll have a few cross-team activities to help you learn from other open source projects.
+If you're new to doc sprints, we can help facilitate.
+If you're experienced, the reigns are yours.
```

Comment:
I think it is "reins" with no 'g' -- right? As in reins for steering a horse?

Reply:
Right, thanks. Horses have reins. Kings have reigns.